test/ecto/multi_test.exs

```diff
@@ -327,6 +327,29 @@ defmodule Ecto.MultiTest do
     ] = Ecto.Multi.to_list(multi)
   end
 
+  test "inspect/1" do
+    name = :halo
+    value = "statue"
+
+    defmodule TestIO do
```

Comment:
I wasn't super sure the best way to test this so I just went with this. Happy to change it though!